website/README.md

```diff
@@ -21,16 +20,14 @@ Alternatively, for development, read the `homepage` section below:
 
 ## The `homepage` docker
 
-The homepage folder builds the static content of the website into standalone files, served by nginx in docker. For development purposes, instead of running docker to test changes, follow these steps in the `gradio/website/homepage` folder.
+The homepage folder builds the static content of the website into standalone files, served by nginx in docker. For development purposes, instead of running docker to test changes, just run `sh scripts/launch_website.sh` from the root of the repo. The steps in that script are the following: 
 
 - `npm install`
-- `npm run build` (or `npm run build-mac` on Mac OSX)
+- `npm run build --url=https://gradio-main-build.s3.amazonaws.com/XXX/`
```

Comment:
Just remove this? I don't think we need to explicitly list these out since users can just read the script. Better to have a single source of truth

Reply:
Good point